bob/src/cluster/quorum.rs

```diff
@@ -143,6 +146,7 @@ impl Quorum {
         mut failed_nodes: Vec<NodeName>,
         key: BobKey,
         data: &BobData,
+        affected_replicas_by_node: &HashMap<NodeName, usize>
```

Comment:
`affected_replicas_by_node` is not needed for aliens, because right now there is no way to save multiple replicas in same alien folder. That's because we have `node_name` -> `vdisk` folder structure